library/include/modules/EventManager.h

```diff
@@ -40,6 +41,7 @@ namespace DFHack {
             typedef void (*callback_t)(color_ostream&, void*); //called when the event happens
             callback_t eventHandler;
             int32_t freq;
+            int32_t when;
```

Comment:
Also asked on [Discord](https://discord.com/channels/793331351645323264/807443262704058408/881243513726074910), but I am curious what the purpose of this field is (is it `tick` or an alternative to `freq`? perhaps a comment would be useful)

Reply:
It's just the global tick count when it the event needs to trigger. iirc the code was dual purposing freq